src/Utils/BreakingChangesFinder.php

```diff
@@ -41,14 +42,14 @@ class BreakingChangesFinder
     public const BREAKING_CHANGE_ARG_CHANGED_KIND              = 'ARG_CHANGED_KIND';
     public const BREAKING_CHANGE_REQUIRED_ARG_ADDED            = 'REQUIRED_ARG_ADDED';
     public const BREAKING_CHANGE_REQUIRED_INPUT_FIELD_ADDED    = 'REQUIRED_INPUT_FIELD_ADDED';
-    public const BREAKING_CHANGE_INTERFACE_REMOVED_FROM_OBJECT = 'INTERFACE_REMOVED_FROM_OBJECT';
+    public const BREAKING_CHANGE_IMPLEMENTED_INTERFACE_REMOVED = 'IMPLEMENTED_INTERFACE_REMOVED';
     public const BREAKING_CHANGE_DIRECTIVE_REMOVED             = 'DIRECTIVE_REMOVED';
     public const BREAKING_CHANGE_DIRECTIVE_ARG_REMOVED         = 'DIRECTIVE_ARG_REMOVED';
     public const BREAKING_CHANGE_DIRECTIVE_LOCATION_REMOVED    = 'DIRECTIVE_LOCATION_REMOVED';
     public const BREAKING_CHANGE_REQUIRED_DIRECTIVE_ARG_ADDED  = 'REQUIRED_DIRECTIVE_ARG_ADDED';
     public const DANGEROUS_CHANGE_ARG_DEFAULT_VALUE_CHANGED    = 'ARG_DEFAULT_VALUE_CHANGE';
     public const DANGEROUS_CHANGE_VALUE_ADDED_TO_ENUM          = 'VALUE_ADDED_TO_ENUM';
-    public const DANGEROUS_CHANGE_INTERFACE_ADDED_TO_OBJECT    = 'INTERFACE_ADDED_TO_OBJECT';
+    public const DANGEROUS_CHANGE_IMPLEMENTED_INTERFACE_ADDED  = 'IMPLEMENTED_INTERFACE_ADDED';
```

Comment:
Sadly, changing those constant is a subtle breaking change.

Reply:
Would you accept an alternative:

Add the new name and use it in our internal code. However, change the value of the old name to the new name and add a deprecation notice for the use of that variant.

This allows existing 14.x consumers to start using the new constant and should not break any existing users of the current constant (if they're using matching the string literal rather than the content then they're already violating the API, since that should be considered an internal representation).

Then the only change to make in 15.0 is to remove the old constant. Any consumer that is already using the new constant then doesn't need to make any additional changes and is compatible with 14.x and 15.0.

Reply:
I've added my proposed change in a new commit :)

Reply:
Good idea 👍 

But I've seen cases where values of similar constants were used as translation keys. So while technically it is an implementation detail it also easily leaks outside.

Anyway, I think it is fine for a minor version as long as we mention it in the changelog. I can live with that.